cpp/StrangeCharacters/test/CharactersTest.cpp

```diff
@@ -0,0 +1,63 @@
+#include "Character.hpp"
+#include "CharacterFinder.hpp"
+
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+
+namespace Characters::Test {
+
+std::vector<std::unique_ptr<Character>> allTestData()
+{
+    std::vector<std::unique_ptr<Character>> characters;
+    characters.emplace_back(std::make_unique<Character>("Joyce", "Byers"));
+    characters.emplace_back(std::make_unique<Character>("Jim", "Hopper"));
+    characters.emplace_back(std::make_unique<Character>("Mike", "Wheeler"));
+    characters.emplace_back(std::make_unique<Character>("Eleven"));
+    characters.emplace_back(std::make_unique<Character>("Dustin", "Henderson"));
+    characters.emplace_back(std::make_unique<Character>("Lucas", "Sinclair"));
+    characters.emplace_back(std::make_unique<Character>("Nancy", "Wheeler"));
+    characters.emplace_back(std::make_unique<Character>("Jonathan", "Byers"));
+    characters.emplace_back(std::make_unique<Character>("Will", "Byers"));
+    characters.emplace_back(std::make_unique<Character>("Karen", "Wheeler"));
+    characters.emplace_back(std::make_unique<Character>("Steve", "Harrington"));
+    characters.emplace_back(std::make_unique<Character>("Mindflayer", std::nullopt, true));
+    characters.emplace_back(std::make_unique<Character>("Demagorgon", std::nullopt, true));
+    characters.emplace_back(std::make_unique<Character>("Demadog", std::nullopt, true));
+    auto& joyce = characters[0];
```

Comment:
this looks clumsy to me, can you declare joyce as a variable first and add it to the list afterwards, like you do in the other test?